### ISSUE TITLE
Missing dropdown to show multiple existing PVs or storage classes while creating a PVC 

### ISSUE THREAD

**Issue by anupama2501** [rancher/dashboard#7270]
**Rancher Server Setup**
- Rancher version: `v2.7-head 40932d6 `
- Installation option (Docker install/Helm Chart): `Docker install`
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):
- Proxy/Cert Details: self signed

**Additional context**
_1. Verified the issue is not seen on v2.6.9 fr RKE2 clusters node drivers.
2. Verified the dropdown works fine for RKE1 node driver clusters_

**Information about the Cluster**
- Kubernetes version: `v1.24.4+rke2r1`
- Cluster Type (Local/Downstream): Downstream Infrastructure provider RKE2 Linode provider 1 node all roles
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions: Cluster Owner
   
**Describe the bug**
While creating a PVC when we select a volume claim a drop down to show existing pvs or storage classes is missing. 

**To Reproduce**
1. Create an rke2 cluster with the specs mentioned above
2. From cluster explorer >> from left navigation >> navigate to Storage >> Persistent volumes >> Create a PV by selecting a volume plugin [I tried with local]
3. Provide the plugin configuration based on the plugin selected above [Since I selected local >> updated the path as /mnt]
4. Add in customize options [Since I selected local, added a node selector]
5. Create the PV
6. From left navigation >> Storage >> select PersistentVolumeClaims >> Under Volume Claim Source select either option [Storage class or existing Persistent Volume] 

**Result**
Notice the dropdown is missing while we want to select existing PVs or Storage classes

**Expected Result**
Expected the dropdown to show up

**Screenshots**
<img width="1000" alt="PVs" src="https://user-images.githubusercontent.com/38144301/197760009-7aacb21a-2965-490f-95c6-5a1d3b803bd7.png">

Noticed while validating https://github.com/rancher/dashboard/issues/6962

**Comment by aalves08** [comment]
@anupama2501 I cannot seem to reproduce this issue... Let me know if you want to jump into a call 🙏 
**NOTE:** rancher/dashboard latest `master` is still 2.7.0 right now

`rancher/rancher:v2.7.0-rc4`, using local cluster `v1.24.4+k3s1`, rancher/dashboard latest `master`
<img width="2050" alt="Screenshot 2022-10-25 at 14 48 57" src="https://user-images.githubusercontent.com/97888974/197792955-3c4db6ca-349c-4b00-936a-5db0490b86ac.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 49 02" src="https://user-images.githubusercontent.com/97888974/197792977-49cc4aa4-15fa-4713-a524-59e10ae60dc0.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 49 08" src="https://user-images.githubusercontent.com/97888974/197792985-d95de873-e881-4066-8cb4-2115e3041d34.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 49 11" src="https://user-images.githubusercontent.com/97888974/197792996-a0523b0b-8ea6-4fb5-b4f3-adf34d7e4ace.png">

`rancher/rancher:v2.7.0-rc4`, using Digital Ocean RKE2 cluster `v1.24.7+rke2r1`, rancher/dashboard latest `master`
<img width="2050" alt="Screenshot 2022-10-25 at 14 54 40" src="https://user-images.githubusercontent.com/97888974/197793036-6f051c73-288e-4172-877d-f18c84481f51.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 55 04" src="https://user-images.githubusercontent.com/97888974/197793046-b090a90a-8c54-4dda-855f-5b30ed1cdf43.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 55 10" src="https://user-images.githubusercontent.com/97888974/197793047-362c4275-47fa-4716-98d2-b1cb3c4e2923.png">
<img width="2050" alt="Screenshot 2022-10-25 at 14 55 13" src="https://user-images.githubusercontent.com/97888974/197793052-2a988727-037d-4167-a1d5-0f8c2cee0113.png">

**Comment by anupama2501** [comment]
@aalves08 apologies, I seemed to have missed your comment on the ticket.

I re validated on v2.7-head 833b1ab

1. Created an rke2 node driver Linode 1 node all roles cluster
2. From explorer >> Storage >> PV -- Created 2 PVs
3. From PVCs dropdown works fine and was able to create a PVC from the existing PVs as well as a default storage class.
<img width="600" alt="rke2pvc" src="https://user-images.githubusercontent.com/38144301/198297007-aa3bc27c-955e-4710-8489-cd670fc43a38.png">

Closing the issue and will open a new issue if seen again in future.
